appveyor.yml

```diff
@@ -1,6 +1,8 @@
 # appveyor file
 # http://www.appveyor.com/docs/appveyor-yml
 
+image: Visual Studio 2017
```

Comment:
Previously we were using the default image, "Visual Studio 2015", which was getting outdated. The old image ran on MS Windows Server 2012, whereas the new image runs on MS Windows Server 2016, which has better support for RDP (remote desktop) that was useful for debugging / evaluating the shell solutions (MSYS2).